service/src/test/java/uk/gov/hmcts/reform/fpl/service/RespondentServiceTest.java

```diff
@@ -134,8 +101,11 @@ void shouldAddPartyIDAndPartyTypeValuesToMultipleRespondents() {
 
     @Test
     void shouldNotAddPartyIDAndPartyTypeValuesToDataStructureIfRespondents1IsNotPresent() {
+        Map<String, Object> data = new HashMap<>();
+        data.put("respondent", "data");
+
         CaseDetails caseDetails = CaseDetails.builder()
-            .data(createData("respondent", "data"))
+            .data(data)
             .build();
 
         AboutToStartOrSubmitCallbackResponse response = service.addHiddenValues(caseDetails);
```

Comment:
Maybe that should invoke the expand method? Doesn't look like anything in the test is invoking it, thus it is not tested properly.